src/sentry/integrations/client.py

```diff
@@ -139,6 +140,9 @@ def _request(self, method, path, headers=None, data=None, params=None,
         if allow_redirects is None:  # is still None
             allow_redirects = method.upper() == 'GET'
 
+        if timeout is None:
+            timeout = 30
```

Comment:
Is this lower than our app server worker timeouts?

Reply:
I'm not sure – I'm guessing our server timeouts are well below 30? I used this as a super liberal fallback. Maybe it's better to just leave the timeout out, unless explicitly passed.

Reply:
I mean, 30 seconds is a pretty ridiculous HTTP request timeout anyways. I'd argue that you most likely need 5 seconds at most. Even for RTT around the world, is significantly less than that.

Reply:
I guess you're also doing this in a loop for each integration. So that'd be really bad. Can we avoid making HTTP requests in a loop in the API endpoint?

Reply:
afaict, `requests` doesn't have a timeout set by default – this isn't any different/worse, imo. I made it a ridiculous number intentionally to effectively match the current non-existent timeout.